clients/admin-ui/src/features/system/SystemInformationFormExtension.tsx

```diff
@@ -0,0 +1,115 @@
+import { Box, FormLabel, Stack } from "@fidesui/react";
```

Comment:
this is mostly just a rename from `DescribeSystemsFormExtension`, except some of the fields (tags and dependencies) got moved to the extension from the base